app/celer-sim/RunnerInput.cc

```diff
@@ -293,11 +294,14 @@ inp::StandaloneInput to_input(RunnerInput const& ri)
             [](inp::PrimaryGenerator const& pg) { return pg.num_events; },
             [](inp::SampleFileEvents const& sfe) { return sfe.num_events; },
             [](inp::ReadFileEvents const& rfe) {
-                if (ends_with(rfe.event_file, ".root"))
+                if (ends_with(rfe.event_file, ".json"))
```

Comment:
These are actually `jsonl` files (one event per line) so the extension should also be `jsonl`.